pkg/monitoring/rules/alerts/prometheus.go

```diff
@@ -0,0 +1,51 @@
+package alerts
+
+import (
+	"errors"
+	"fmt"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"os"
+	"strings"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+)
+
+const (
+	prometheusRunbookAnnotationKey = "runbook_url"
+	partOfAlertLabelKey            = "kubernetes_operator_part_of"
+	partOfAlertLabelValue          = "kubevirt"
+	componentAlertLabelKey         = "kubernetes_operator_component"
+	componentAlertLabelValue       = "ssp-operator"
+	defaultRunbookURLTemplate      = "https://kubevirt.io/monitoring/runbooks/%s"
+	runbookURLTemplateEnv          = "RUNBOOK_URL_TEMPLATE"
+)
+
+func Register() error {
+	alerts := operatorAlerts()
+
+	runbookURLTemplate, err := getRunbookURLTemplate()
+	if err != nil {
+		return err
+	}
+
+	for _, alert := range alerts {
+		alert.Labels[partOfAlertLabelKey] = partOfAlertLabelValue
+		alert.Labels[componentAlertLabelKey] = componentAlertLabelValue
+		alert.Annotations[prometheusRunbookAnnotationKey] = fmt.Sprintf(runbookURLTemplate, alert.Alert)
+	}
+
+	return operatorrules.RegisterAlerts([][]promv1.Rule{alerts}...)
```

Comment:
It is not needed to create a slice of slices here. You can simply pass the `alerts` slice to the function:
```go
return operatorrules.RegisterAlerts(alerts)
```